x-pack/plugins/reporting/server/routes/index.ts

```diff
@@ -18,6 +18,7 @@ import { registerLegacy } from './legacy';
 export function registerRoutes(server: KbnServer) {
   const config = server.config();
   const DOWNLOAD_BASE_URL = config.get('server.basePath') + `${API_BASE_URL}/jobs/download`;
+  const hasPanelActionCSVEnabled = config.get('xpack.reporting.csv.enablePanelActionDownlad');
   const { errors: esErrors } = server.plugins.elasticsearch.getCluster('admin');
```

Comment:
Lines 20-22 are bothering me because the variables are not defined near the code they are used. Would you mind doing a quick touch-up so they're initialized right above where they're needed?